libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -980,6 +987,16 @@ bool RC_Channel::read_3pos_switch(RC_Channel::aux_switch_pos_t &ret) const
     return true;
 }
 
+// return switch position value as LOW, MIDDLE, HIGH
+// if reading the switch fails then it returns LOW
+RC_Channel::aux_switch_pos_t RC_Channel::get_aux_switch_pos() const
+{
+    RC_Channel::aux_switch_pos_t position = aux_switch_pos_t::LOW;
```

Comment:
```suggestion
    aux_switch_pos_t position = aux_switch_pos_t::LOW;
```

This is a niggle and doesn't really need to be fixed.

Reply:
Thank you! 
It should be fixed.